README.md

```diff
@@ -1,5 +1,347 @@
+of each page. And the gray banner contains a way for the end user to switch in
+to and out of the Next.js preview mode.
+
+Note that within Storyblok, there is no button to switch out of preview mode.
+This was done by design because the whole point of the Storyblok UI is to be
+able to preview content that hasn't been published yet.
+
+## System design decisions
+
+Each website should have only one project in Vercel. Having more than one
+project results in several preview URLs being posted to each pull request on
+GitHub, which makes it hard for reviewers to know which preview URL they should
+review. With only one project per website, you get one preview URL (per website)
+posted to the pull request. If it's not clear from the pull request title or
+code which website the PR affects, the author should clarify in the PR
+description. Ideally, the PR author should add one or both of the (`Labs`,
+`LLC`) GitHub labels, as appropriate.
+
+Code in the `main` branch should only be used by the production Vercel
+environment, and it should show only published content. While it may be tempting
+to want to see draft content against the `main` branch code, it creates the
+potential for confusion. For example, if someone takes a screenshot of a
+webpage, and in the screenshot you can see in the browser address bar that the
+URL is on the live production site (such as quansight.com), then you will be
+able to safely assume that you're seeing content that was **published** in
+Storyblok at the time the `main` branch was deployed to production. The other
+reason for this discipline is that it's better to limit reviewers and content
+editors to previewing draft content against the `develop` branch in order to
+help catch any potential code/content conflicts before merging to `main`.
+
+When viewing the site in local development or at a Vercel preview URL, there
+should be a banner that explains that you are looking at a preview of the site.
+This helps reduce confusion when screenshots are shared. This banner should not
+be present on a production build of the site.
+
+The preview/development environment banner should allow the user to toggle
+between seeing published versus draft/saved Storyblok content. The banner should
+change colors to indicate which of the two content preview modes the user is in.
+
+The above should hold _except_ when the user is using the Storyblok web
+interface; then the website should always be in "draft" mode, showing saved (but
+not published) content. And as mentioned previously, it should show this content
+against the latest code from the `develop` branch (not the main branch).
+Related: the content team shouldn't have to think about which URL to use with
+the Storyblok editor. It should be just one default URL, and this URL should
+show draft content against the latest code in `develop`.
+
+When a reviewer visits a Vercel preview URL, the site should show draft content
```

Comment:
This seems inconsistent with the table above

Reply:
It is. Thanks for the catch. I'm deleting this paragraph for now. Might need to revisit this decision later.